code/core/src/manager-api/lib/open-in-editor.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Open the file in the editor
+ *
+ * Available for builders which support https://github.com/yyx990803/launch-editor
+ *
+ * Known builders: Webpack5, Vite
+ *
+ * @param filePath - The path to the file to open in the editor
+ * @returns Void
+ */
+export async function openInEditor(
+  filePath: string,
+  line?: number,
+  column?: number
+): Promise<void> {
+  let fileLocation = filePath;
+  if (typeof line === 'number') {
+    fileLocation += `:${line}`;
+    if (typeof column === 'number') {
+      fileLocation += `:${column}`;
+    }
+  }
+
+  try {
+    await fetch(`/__open-in-editor?file=${encodeURIComponent(fileLocation)}`, {
+      method: 'POST',
+    });
```

Comment:
Can you explain the reasoning behind using a custom URL and fetch, rather then the channel?